modules/oagw/oagw-gw/src/module.rs

```diff
@@ -0,0 +1,141 @@
+//! OAGW Gateway module definition.
+
+use std::sync::Arc;
+
+use async_trait::async_trait;
+use modkit::api::OpenApiRegistry;
+use modkit::{DbModule, Module, ModuleCtx, RestfulModule};
+use modkit_security::SecurityCtx;
+use oagw_sdk::{OagwApi, OagwPluginSpecV1};
+use sea_orm_migration::MigratorTrait;
+use tracing::info;
+use types_registry_sdk::TypesRegistryApi;
+
+use crate::api::rest::routes;
+use crate::config::OagwConfig;
+use crate::domain::ports::StubSecretResolver;
+use crate::domain::repo::{LinkRepository, RouteRepository};
+use crate::domain::service::{Service, ServiceConfig};
+use crate::infra::storage::link_repo::SeaOrmLinkRepository;
+use crate::infra::storage::route_repo::SeaOrmRouteRepository;
+use crate::local_client::OagwLocalClient;
+
+/// OAGW Gateway module.
+///
+/// This module provides:
+/// - Route and link configuration management
+/// - Plugin-based protocol and authentication handling
+/// - REST API for outbound API invocation
+#[modkit::module(
+    name = "oagw_gw",
+    deps = ["types_registry", "oagw_default_plugin"],
+    capabilities = [db, rest]
+)]
+pub struct OagwGateway {
+    service: arc_swap::ArcSwapOption<Service>,
+}
+
+impl Default for OagwGateway {
+    fn default() -> Self {
+        Self {
+            service: arc_swap::ArcSwapOption::from(None),
+        }
+    }
+}
+
+#[async_trait]
+impl Module for OagwGateway {
+    async fn init(&self, ctx: &ModuleCtx) -> anyhow::Result<()> {
+        info!("Initializing OAGW gateway module");
+
+        // Load module configuration
+        let cfg: OagwConfig = ctx.config()?;
+
+        // Acquire DB with SecureConn for security-aware queries
+        let db = ctx.db_required()?;
+        let sec_conn = db.sea_secure();
+
+        // === SCHEMA REGISTRATION ===
+        // Gateway registers the plugin SCHEMA; plugins register their INSTANCES
+        let registry = ctx.client_hub().get::<dyn TypesRegistryApi>()?;
+
+        // Register OAGW plugin schema
+        let schema_str = OagwPluginSpecV1::gts_schema_with_refs_as_string();
```

Comment:
Accrosing to intent scenarios (5.1 Gateway Module Initialization), we should register:
gts.x.core.oagw.proto.v1~
gts.x.core.oagw.stream_proto.v1~
gts.x.core.oagw.auth_type.v1~
and Load well-known instances from gts/*.instances.json.

But I don't see it here.